app/models/fee/base_fee_type.rb

```diff
@@ -32,6 +32,10 @@ class BaseFeeType < ApplicationRecord
     include CaseUpliftable
     include DefendantUpliftable
 
+    # TODO: update all agfs_scheme_10 fee types to also have agfs_scheme_12 roles
```

Comment:
Is there a timeframe for completing this?

Reply:
am working on the "proper" fix now.

Reply:
i cherry-pick'ed this out of the [proper fix PR](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3512)